.github/workflows/workspace.yaml

```diff
@@ -1,37 +1,37 @@
 name: Workspace
```

Comment:
Let's rename this workflow to be clearer. Maybe even rename the file.
```suggestion
name: Lint
```

Reply:
renamed workflow and file